tests/parsing/grammar/expressions/expected/es6template.res.txt

```diff
@@ -90,13 +90,13 @@ let s =
 |js})
   [@res.template ])
 let s = (({js|$dollar without $braces $interpolation|js})[@res.template ])
-let s = (({json|null|json})[@res.template ])
+let s = ((json [|(("null")[@res.template ])|] [||])[@res.taggedTemplate ])
```

Comment:
This needs fixing too.

Reply:
This is accurate as well.  The old code wasn't setting `~attrs:[templateLiteralAttr]` on all nodes generated by a template string, but the new code is.

Reply:
printer/expr/expected/templateLiteral.res.txt has the following line:
```
let x = json`null`
```
which indicates to mean that things are working as expected.